samples/LiveEffect/src/main/cpp/CMakeLists.txt

```diff
@@ -39,6 +39,8 @@ target_link_libraries(liveEffect
         atomic
         log)
 
+# Enable optimization for release build; it could be enabled for debug build
+# if performance is of interest.
 target_compile_options(liveEffect
     PRIVATE
-        -Wall -Werror)
+        -std=c++14 -Wall -Werror $<$<CONFIG:RELEASE>:-Ofast>)
```

Comment:
Why add -std=c++14? I don't think the app needs any C++14 features in which case it makes sense to leave this out.

Reply:
done